packages/nimble-components/src/card/tests/card.stories.ts

```diff
@@ -0,0 +1,40 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { loremIpsum } from '../../utilities/tests/lorem-ipsum';
+import {
+    createUserSelectedThemeStory,
+    incubatingWarning
+} from '../../utilities/tests/storybook';
+import { buttonTag } from '../../button';
+import { cardTag } from '..';
+
+const overviewText = `The \`nimble-card\` is a container that is designed to contain arbitrary content that is specified by a client
```

Comment:
I modeled this overview text off of the nimble-card-button's overview. Do we typically have a tech writer review these strings?

Reply:
No, we've never gotten a tech writer to review these descriptions. A lot of time we'll pull text from the W3C description of what a specific component is, but I don't think that applies here. I think what you have here is fine, and I'm sure other reviewers will feel free to weigh in if they have changes they'd like to see.